Remove platform dependency expressions that require PEP426 support we don't
have yet.  Not suitable for upstream.

--- ORIGINAL/requirements.txt	2015-12-29 14:23:49.000000000 -0800
+++ cloudbase-init-0.9.9/requirements.txt	2015-12-30 08:13:34.146913324 -0800
@@ -1,4 +1,4 @@
-pbr>=1.3
+pbr>=0.8.1
 iso8601
 eventlet
 netaddr>=0.7.6
@@ -11,7 +11,3 @@
 netifaces
 PyYAML
 requests
-pywin32;sys_platform=="win32"
-comtypes;sys_platform=="win32"
-wmi;sys_platform=="win32"
-tzlocal;sys_platform=="win32"
